File: 37. 反转一个3位整数.cpp
class Solution {
public:
    /**
     * @param number: A 3-digit number.
     * @return: Reversed number.
     */
    int reverseInteger(int number) {
        int a, b, c;
        a = number % 10;
        number /= 10;
        b = number % 10;
        number /= 10;
        c = number;
        return a * 100 + b * 10 + c;
    }
};

// 总耗时 77 ms
// 您的提交打败了 46.60% 的提交!

File: Python3/37. 反转一个3位整数.py
class Solution:
    """
    @param number: A 3-digit number.
    @return: Reversed number.
    """
    def reverseInteger(self, number):
        ret = 0;
        time = 100;
        for i in range(3) :
            ret += time * (number % 10);
            number //= 10;
            time //= 10;
        return ret

# 总耗时 503 ms
# 您的提交打败了 63.60% 的提交!